src/PrestaShopBundle/Entity/FeatureFlag.php

```diff
@@ -125,7 +125,7 @@ public function getName(): string
     /**
      * @return bool
      */
-    public function getState(): bool
+    public function isEnabled(): bool
     {
         return $this->state;
```

Comment:
Sorry about this 😅 But since you now use isEnabled everywhere I think the schema itself should be renamed So the column should be renamed to `enabled` or `isEnabled` as well

Reply:
A) **4 more PRs are expected to complete this feature (see PR body)**

- When user installs 1.7.8.0 or upgrades to 1.7.8.0, inject into database the Product Page Feature flag

- Make sure all wordings are translatable, including feature flag names and descriptions

- Modal improvements

  - Allow confirmation modal to adapt the wording depending if it's plural or singular
  - Only show the modal when flags are enabled, not disabled

- Link the Product Page V2 flag to V2 Page Product

_Can I schedule this improvement for a 5th PR, so we dont block the progress of this PR?_

B) Can you please have a discussion with other maintainers so I have a group approval for the name `enabled` 😛  I'm afraid to see another battle on the field name, and that would again block this PR. I just want to know that, when I change the field name and the mapping, I do this for the last time 😊  .